src/content/en/2020/markup.md

```diff
@@ -457,22 +483,58 @@ Yet the most popular attribute is `class`—with almost 3 billion occurrences in
 | `rel` | 171,802,460 | 1.96% |
 | `value` | 140,666,779 | 1.61% |
 
-<figcaption>{{ figure_link(caption="Top 10 attributes.", sheets_gid="1348855449", sql_file="pages_almanac_by_device_and_attribute_name_frequency.sql") }}</figcaption>
+<figcaption>{{ figure_link(caption="Top 10 attributes by frequency of use.", sheets_gid="1348855449", sql_file="pages_almanac_by_device_and_attribute_name_frequency.sql") }}</figcaption>
 </figure>
 
-The `value` attribute, which specifies the value of an `input` element, surprisingly completes the top ten—surprising because on our end, subjectively, we didn’t get the impression values were used that frequently.
+The most popular attribute is `class`, with nearly 3 billion occurrences in our dataset and constituting 34% of all attributes in use. `class` is by far the most prevalent attribute.
+
+The `value` attribute, which specifies the value of an `input` element, surprisingly completes the top ten. It's surprising to us because, subjectively, we didn't get the impression values were used that frequently.
 
 ### Attributes on pages
 
-Are there attributes that we find in every document? No—but almost. `href` (99.21%), `src` (99.18%), `content` (98.88%), `name` (98.61%), `type` (98.55%), `class` (98.24%), `rel` (97.98%), `id` (97.46%), `style` (95.95%), and `alt` (90.75%) make the Top 10 here, with each being found on at least 90% of sampled pages.
+Are there attributes that we find in every document? Not quite, but almost:
 
-This raises questions we cannot answer like yes, `type` is used on other elements, too, but why this tremendous popularity, given that it’s usually not needed to specify for style sheets or scripts (with CSS and JavaScript being assumed default)? Or how do we really fare with `alt`, do those 9.25% of pages that don’t include it not contain any images?
+<figure markdown>
```

Comment:
The original draft was deliberate about text form as the information here didn’t seem that critical. If you object against the text form, please shorten this table to 10 attributes—it’s very little value for this much space otherwise.